addons/crm/tests/test_crm_lead.py

```diff
@@ -112,6 +117,7 @@ def test_crm_lead_creation_partner(self):
         self.assertEqual(lead.city, self.contact_1.city)
         self.assertEqual(lead.zip, self.contact_1.zip)
         self.assertEqual(lead.country_id, self.contact_1.country_id)
+        self.assertEqual(lead.lang_id.code, self.contact_1.lang)
```

Comment:
Isn't it checking False versus False ? Lang does not seem set in setup of crm.

Reply:
No? I printed the values and they are `en_US` for `lead.lang_id.code` and `en_US` for `self.contact_1.lang`.

`lang_id` (not `lang`) is one of the fields of `crm.lead`

Reply:
Ok, thought it was not set. Actually indeed there is a default value for lang on partner which is self.env.lang or _context.get('tz'). Probably worth a check, like assert that self.contact_1.lang is en_US before the test as this is the expected default value.